docs/versioned_docs/version-2.x/components.mdx

```diff
@@ -209,13 +213,18 @@ word vectors in your pipeline.
       # Name of the language model to use
       model_name: "bert"
       # Pre-Trained weights to be loaded
-      model_weights: "bert-base-uncased"
+      model_weights: "rasa/LaBSE"
 
       # An optional path to a specific directory to download and cache the pre-trained model weights.
       # The `default` cache_dir is the same as https://huggingface.co/transformers/serialization.html#cache-directory .
       cache_dir: null
   ```
 
+  :::caution Changed in 2.1
+  Pre-trained weights have been renamed from `bert-base-uncased` to `rasa/LaBSE`.
```

Comment:
I think I've still used `bert-base-uncased` in a later version. And it shows as a potential model weight from `transformers` when `rasa/laBSE` doesn't. So my preference would be to remove this :)

Reply:
made a more specific comment in this commit: 75765a41ff1

AFAIK it's what we document in the latest version of the doc [here](https://rasa.com/docs/rasa/components#hftransformersnlp). So maybe the message I wrote is not correct? What would you suggest?

Reply:
yeah I had trouble with this before haha. Because you can still use other pre trained weights, these are just the ones we suggest.. and saying that they've been renamed implies to me that it's your only option when it's just our recommended one... so I'd suggest removing this "changed in" and leaving the section as is

Reply:
🤔 I still think we need to mention this to the user somehow, because it's a breaking change from their POV. Wondering if the confusion you had is because you're a power user of our docs / products? I've attempted another commit 0c7557c5f6b to make it more explicit. It looks like this:
```
  :::caution Changed in 2.1
  Pre-trained weights for BERT have been renamed from `bert-base-uncased` to `rasa/LaBSE`.
  Other types of language model have been left unchanged.
  :::
```
WDYT?

Reply:
made a suggestion below that I think settles it, idk why I didn't think of this before lol